src/account/Settings.test.tsx

```diff
@@ -108,13 +108,8 @@ describe('Account', () => {
   })
 
   it('navigates to PincodeSet screen if entered PIN is correct', async () => {
-    const store = createMockStore({
-      app: {
-        showGuidedOnboardingCopy: true,
-      },
-    })
```

Comment:
I actually think this test would be a bit better if this were still here. since it would make sure that even if the feature flag is set to true, you don't get the onboarding copy if you're coming from settings.

without the feature flag set to true though, the test doesn't seem to help verify that behavior

Reply:
not important though